fhttp/http_utils.go

```diff
@@ -449,3 +449,29 @@ func NewHTMLEscapeWriter(w io.Writer) io.Writer {
 func OnBehalfOf(o *HTTPOptions, r *http.Request) {
 	_ = o.AddAndValidateExtraHeader("X-On-Behalf-Of: " + r.RemoteAddr)
 }
+
+const (
```

Comment:
we have those constants a few places already, notably fgrpc/grpcrunner.go - I guess we can clean that up later / put them in a common place without adding dependencies (maybe fnet/)